### PR TITLE
[docs][image-picker] Added missing platform spec. to exif-property

### PR DESCRIPTION
**Pull request by entiendoNull**
# Why

The web version of the expo-image-picker package does not currently have an Exif-implementation. Until one is added the docs should be updated accordingly in order to specify where this functionality is available.

# How

I updated the reference docs as described here;
https://github.com/expo/expo/blob/main/docs/README.md#update-api-reference-docs 

# Test Plan

Before:
image

After:
image
(Updated the image after receiving feedback about platform order) 

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: packages/expo-image-picker/src/ImagePicker.types.ts [hunk 1]
```diff
@@ -261,6 +261,9 @@ export type ImagePickerAsset = {
    * The `exif` field is included if the `exif` option is truthy, and is an object containing the
    * image's EXIF data. The names of this object's properties are EXIF tags and the values are the
    * respective EXIF values for those tags.
+   *
+   * @platform ios
+   * @platform android
```
Comment (amandeepmittal) [inline packages/expo-image-picker/src/ImagePicker.types.ts:266]:
```suggestion
   * @platform android
   * @platform ios
```

As per our writing styleguide, we prefer to starting Platform order from Android when referencing multiple platforms: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#platform-order-when-referencing-android-ios-and-web-or-multiple-platforms

Can you please swap their order?

**Review by amandeepmittal — CHANGES_REQUESTED** [review]

File: packages/expo-image-picker/src/ImagePicker.types.ts (hunk 1 above)
Comment (entiendoNull) [reply packages/expo-image-picker/src/ImagePicker.types.ts:266]: Ops, sorry about that. I copied it directly from another property in the same file. Would you like to change this at the same time?
image

Comment (entiendoNull) [reply packages/expo-image-picker/src/ImagePicker.types.ts]: I swapped the order of supported platforms of the `assetId` property as well while at it 👍

**Comment by entiendoNull** [comment]
iOS build failed due to `Uncaught Error Error: ENOSPC: no space left on device`. 

Might need to run that again 😊

**Comment by entiendoNull** [comment]
@amandeepmittal, iOS build failed due to Uncaught Error Error: ENOSPC: no space left on device.

Might need to run that again 😊

**Review by amandeepmittal — APPROVED** [review]
Thanks for fixing the platform order outdated issue with other properties! 

Docs changes are 👍

**Comment by entiendoNull** [comment]
> Thanks for fixing the platform order outdated issue with other properties!
> 
> Docs changes are 👍

My pleasure - and yes, the Expo docs are fantastic! 
Happy to contribute 👍

**Comment by entiendoNull** [comment]
@amandeepmittal, just a heads up that the checks have passed.

**Pull request merged**